src/components/App.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-useless-constructor */
+import React from 'react';
+import CalculatorDisplay from './Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] [OPTIONAL] Please remove `eslinte` comments and try to solve  your problem other way! It will be good for your technical skill.

Reply:
Okay thank you